classes/announcement/PKPAnnouncementDAO.inc.php

```diff
@@ -338,7 +338,7 @@ function &getAnnouncementsNotExpiredByAssocId($assocType, $assocId, $rangeInfo =
 			FROM announcements
 			WHERE assoc_type = ?
 				AND assoc_id = ?
-				AND (date_expire IS NULL OR date_expire > CURRENT_DATE)
+				AND (date_expire IS NULL OR date_expire >= CURRENT_DATE)
 				AND (date_posted < CURRENT_DATE)
```

Comment:
Whoops -- You're absolutely right, @beghelli. I cherry-picked the wrong commit from my development branch for creating the PR. Sorry. 

I'm not sure how I should proceed now. Should I update this PR or create a new one for you guys?